Tasks/ServiceFabricUpdateAppVersions/Update-PackageVersion.psm1

```diff
@@ -77,7 +77,7 @@
                     # Search the package files for differences (Unless the user doesn't want to log all changes and we already know the package xml changed)
                     $newPackagePath = Join-Path $NewPackageRoot $packageName
                     $oldPackagePath = Join-Path $OldPackageRoot $packageName
-                    if (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges)
+                    if ((Test-Path $newPackagePath) -and (Find-FileChanges $newPackagePath $oldPackagePath $LogIndent -LogAllChanges:$LogAllChanges))
```

Comment:
That's Scenario 3 above, right?  In that case, it's not necessary to bump the version.  Compared to the previous version of the application, the config package is a branch new package being added.  So there's no need to change its version from what it was defined as.  The upgrade doesn't care what it's version is since its a brand new package.